Change versionning warning (#594)
meilisearch/documentation: references/README.md

Welcome to the MeiliSearch API documentation.

::: tip

> You might find the answers to some of your questions [in the FAQ](/faq/faq.md) 💡

Reviewer: ```suggestion
Check out [the FAQ](/faq/faq.md) for answers to some common questions 💡
```

:::

## Version Compatibility

> The documentation is compatible with the latest version of MeiliSearch. We do not provide multi-versionning documentation.

Reviewer: ```suggestion
This documentation reflects only the latest version of MeiliSearch. We do not provide multi-version documentation.
```

> The actual version of [MeiliSearch](https://github.com/meilisearch/MeiliSearch) is written on the upper navbar on the left side.

Reviewer: ```suggestion
The current version of [MeiliSearch](https://github.com/meilisearch/MeiliSearch) can be found on the left side of the upper navbar.
```

> If you have a doubt concerning the version of [MeiliSearch](https://github.com/meilisearch/MeiliSearch), it is written on the upper navbar on the left side.

Reviewer 2: "If you have a doubt about the version of Meilisearch" :  I find  "a doubt" very wide, I could have a doubt about the differences between the version I have installed and the latest one.  Or maybe I don't know which version I have installed in my computer. Maybe just change this to something like : "The actual version of Meilisearch is written..."
